guide/payments/contacts.md

```diff
@@ -0,0 +1,229 @@
+    - file: sending-review
+      alt: Invoice screen with a contact assigned
+      caption: Final review of the payment before sending it.
+imagesReceive:
+    - file: receive-home
+      alt: Home screen with amount input
+      caption: The user enters an amount and taps "Request".
+    - file: receive-qr
+      alt: Payment request screen
+      caption: An invoice screen with minimal information.
+    - file: receive-qr-with-contact
+      alt: Payment request screen with a contact assigned
+      caption: The same invoice with a contact assigned.
+imagesActivity:
+    - file: activity
+      alt: Activity screen showing transactions with minimal information
+      caption: By default, payments contain little to no contextual information.
+    - file: activity-annotated
+      alt: Activity screen showing transactions with assigned contacts
+      caption: Assigning contacts to payments makes them much easier to understand and work with.
+    - file: activity-transaction
+      alt: Transaction details
+      caption: Transaction detail screens should have convenient options to assign a contact.
+    - file: sending-pick-contact
+      alt: Screen with a list of contacts to choose from
+      caption: The user picks a contact in a slide-up modal.
+    - file: activity-transaction-annotated
+      alt: Transaction details with assigned contact
+      caption: A payment with an assigned contact.
+
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/contacts/contacts-header.jpg"
+    retina = "/assets/images/guide/payments/contacts/contacts-header@2x.jpg"
+    mobile = "/assets/images/guide/payments/contacts/contacts-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/contacts/contacts-header-mobile@2x.jpg"
+    alt-text = "Illustration of large contact cards"
+    width = 1600
+    height = 400
+    layout = "full-width"
+%}
+
+
+# Contacts
+{:.no_toc}
+
+<!--
+
+
+-->
+
+Whether we’re sending emails, physical mail, or following someone on social media, we primarily think in terms of names and faces, and not the respective address or user ID.
+
+Invoices, node IDs and other transaction endpoints in bitcoin and Lightning are highly unintuitive. Abstracting them via a contact list can create a much smoother user experience. There are many [payment request formats]({{ '/guide/payments/request/payment-request-formats/' | relative_url }}), each with unique properties and varying levels of maturity and adoption, requiring unique design solutions. This page uses the more approachable term "address", along with various UI techniques, to abstract these complexities for users.
+
+Let's go over common user interactions around contacts. This will illustrate how such a feature could work, and helps explain the underlying design problems and decisions.
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
```

Comment:
We need to have a section on the wallet users contact card which I would put first here. This would cover how they can add their own addresses to it and so forth. Maybe it comes with an address already (say if they setup a Lightning address during on boarding). Likely want to cover flows around sharing their contact card as that is likely what many will do with theirs as well.

Reply:
Having a contact card for the owner doesn't seem quite right. When "sharing your contact", don't you want to use the request flow? If a wallet doesn't let you add a Lightning address, would the screen be empty?

Reply:
> Having a contact card for the owner doesn't seem quite right. 

I don't know of any app that uses contacts that does not let the user have their own contact card. 

> When "sharing your contact", don't you want to use the request flow?

Not necessarily, if I have my own contact card and it has say 3 addresses in it I'd rather just share that contact card with someone and they can choose one of the 3. 

How would you share your Lightning address during the request flow? 

> If a wallet doesn't let you add a Lightning address, would the screen be empty?

It would  just start empty if the app didn't have the user setup some kind of address during onboarding or something. It could still just be blank and have the option to add a profile image.